doc/schema/relationship-types/belongs-to.md

```diff
@@ -49,7 +49,7 @@ An attribute named `salesPersonId` will be created as if it were defined as foll
 const Account: PartialSchema = {
   name: "Account",
   attributes: {
-    salesPersonId: uuid({ references: "SalesPerson" }), // 👀
+    salesPersonId: uuid(), // References SalesPerson.id
   },
   relationships: {
     salesPerson: belongsTo(),
```

Comment:
similary for FE relationship typing on stuff like `Account.findAll(accounts => accounts[0].salesPerson.<typescript>)` we need this to be `salesPerson: belongsTo("SalesPerson")`; should our examples show the same? thoughts @nlundquist

Reply:
yeah i mean we can still document that it's optional but i think our examples should show the schema name passed in as a string just so a newbie to hatchify if they follow examples/guides won't find them without types on the frontend exactly

Reply:
i wish we didn't have to do be so explicit on the frontend lol but it's a limitation we have to live with

Reply:
I think we should document the way we'd "like" for it be used. 

We should also try to have a conversation that includes Justin about how we _could_ solve this (and make other problems easier) if we considered generating TS types from the schema sort of like how Prisma does it, instead of trying to do everything completely dynamically.